### PR TITLE
Feature/uob ezpay

### PR DESCRIPTION
**Pull request by uziwuzzy**
- add uob ezpay payment method
- fix  error transaction handling for shopeepay

### CODE REVIEW

File: MidtransCoreKit/MidtransCoreKit/MidtransConstant.h [hunk 1]
```diff
@@ -129,10 +130,10 @@ static NSString *const MIDTRANS_UAT_BIN_URL = @"https://app.stg.midtrans.com/sna
 
 /*staging*/
 
-static NSString *const MIDTRANS_STAGING_API_URL = @"https://api.stg.midtrans.com/v2";
+static NSString *const MIDTRANS_STAGING_API_URL = @"https://midtrans-checkout.stg.veritrans.co.id/v2";
 static NSString *const MIDTRANS_STAGING_MIXPANEL = @"cc005b296ca4ce612fe3939177c668bb";
-static NSString *const MIDTRANS_STAGING_SNAP = @"https://app.stg.midtrans.com/snap/v1";
-static NSString *const MIDTRANS_STAGING_BIN_URL = @"https://app.stg.midtrans.com/snap/v1/bank_bins";
+static NSString *const MIDTRANS_STAGING_SNAP = @"https://midtrans-checkout.stg.veritrans.co.id/snap/v1";
+static NSString *const MIDTRANS_STAGING_BIN_URL = @"https://midtrans-checkout.stg.veritrans.co.id/snap/v1/bank_bins";
```
Comment (askur87) [inline MidtransCoreKit/MidtransCoreKit/MidtransConstant.h:136]: this URL for temporary or permanent we will change to this ?

File: MidtransDemo/MidtransDemo/MDOrderViewController.m [hunk 2]
```diff
@@ -106,7 +106,9 @@ - (void)viewDidLoad {
     [[MidtransNetworkLogger shared] startLogging];
     
     CONFIG.callbackSchemeURL = @"demo.midtrans://";
-    CONFIG.shopeePayCallbackURL = @"demo.midtrans://";
+//    CONFIG.shopeePayCallbackURL = @"demo.midtrans://";
```
Comment (askur87) [inline MidtransDemo/MidtransDemo/MDOrderViewController.m:109]: please remove if not use

File: MidtransKit/MidtransKit/UOB/MIDUobMenuController.m [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (weak, nonatomic) IBOutlet MIdtransUIBorderedView *totalAmountBorderedView;
+@property (weak, nonatomic) IBOutlet UILabel *orderIdLabel;
+@property (nonatomic) NSArray *uobOptionListTitle;
+@property (nonatomic) NSArray *uobOptionListDescription;
+@property (nonatomic) NSArray *uobSelectedOptionTitles;
+@property (nonatomic) NSArray *uobSelectedOptions;
+@property (nonatomic) MIDUobMenuContent *MenuContentApp;
+@property (nonatomic) MIDUobMenuContent *MenuContentWeb;
+//@property (nonatomic) NSArray *MenuContents;
```
Comment (askur87) [inline MidtransKit/MidtransKit/UOB/MIDUobMenuController.m:35]: this one also please remove if not use

File: MidtransCoreKit/MidtransCoreKit/MidtransConstant.h (hunk 1 above)
Comment (uziwuzzy) [reply MidtransCoreKit/MidtransCoreKit/MidtransConstant.h:136]: this is temporary, since the current staging url is using this one.

Comment (askur87) [reply MidtransCoreKit/MidtransCoreKit/MidtransConstant.h:136]: noted

**Review by askur87 — APPROVED** [review]
LGTM

**Pull request merged**